test/objectwrap_function.cc

```diff
@@ -36,7 +36,8 @@ class FunctionTest : public Napi::ObjectWrap<FunctionTest> {
 };
 
 std::unordered_map<napi_env, Napi::FunctionReference>
-    FunctionTest::constructors = {};
+    FunctionTest::constructors =
+        std::unordered_map<napi_env, Napi::FunctionReference>();
```

Comment:
I think just dropping the `=` or the ` = {}` would also do the trick here

Reply:
That's what I just tested out, removing the = {}